core/serialisation/data_serialiser.py

```diff
@@ -160,10 +160,10 @@ class Meta:
 
     submission_id = auto_field(model=Submission, data_key="SubmissionID")
     programme_id = auto_field(model=Programme, data_key="ProgrammeID")
-    question = fields.String(attribute="data_blob.question", data_key="Question")
-    indicator = fields.String(attribute="data_blob.indicator", data_key="Indicator")
-    response = fields.String(attribute="data_blob.response", data_key="Answer")
-    guidance_notes = fields.String(attribute="data_blob.guidance_notes", data_key="GuidanceNotes")
+    question = fields.String(attribute="data_blob.question", data_key="Question", dump_default=None)
+    indicator = fields.String(attribute="data_blob.indicator", data_key="Indicator", dump_default=None)
+    response = fields.String(attribute="data_blob.response", data_key="Answer", dump_default=None)
+    guidance_notes = fields.String(attribute="data_blob.guidance_notes", data_key="GuidanceNotes", dump_default=None)
     programme_name = auto_field(model=Programme, data_key="Place")
```

Comment:
so the behaviour for the jsonb fields now defaults to empty cell if no data present, but does the same thing happen for auto_fields? We should check we have consistent behaviour between these 2 different methods of serialising.

Reply:
I'll have to doublecheck, but I don't think auto_fields has the ```dump_default``` param. The issue was that values in the JSONB are not guaranteed to be present between funds and rounds, and this causes a mismatch of column orderings depending on what data is ingested. All of the auto_field fields are either FKs, PKs, non-nullable, or datetimes, and so (apart from maybe a couple of edge cases in the latter) don't need a default value for the sake of ordering.